src/main/scala/com/campudus/tableaux/verticles/MessagingVerticle/MessagingVerticle.scala

```diff
@@ -0,0 +1,477 @@
+package com.campudus.tableaux.verticles.Messaging
+
+import com.campudus.tableaux.TableauxConfig
+import com.campudus.tableaux.database.DatabaseConnection
+import com.campudus.tableaux.database.domain.ColumnType
+import com.campudus.tableaux.database.domain.Service
+import com.campudus.tableaux.database.domain.ServiceType
+import com.campudus.tableaux.database.domain.Table
+import com.campudus.tableaux.database.model.ServiceModel
+import com.campudus.tableaux.database.model.StructureModel
+import com.campudus.tableaux.database.model.TableauxModel
+import com.campudus.tableaux.database.model.TableauxModel.{ColumnId, RowId, TableId}
+import com.campudus.tableaux.helper.VertxAccess
+import com.campudus.tableaux.router.auth.permission.RoleModel
+import com.campudus.tableaux.router.auth.permission.TableauxUser
+
+import io.vertx.core.buffer.Buffer
+import io.vertx.core.json.JsonObject
+import io.vertx.lang.scala.ScalaVerticle
+import io.vertx.scala.SQLConnection
+import io.vertx.scala.core.Vertx
+import io.vertx.scala.core.eventbus.Message
+import io.vertx.scala.core.http.HttpClient
+import io.vertx.scala.core.http.HttpServer
+import io.vertx.scala.ext.web.client.HttpResponse
+import io.vertx.scala.ext.web.client.WebClient
+import org.vertx.scala.core.json.Json
+
+import scala.collection.JavaConverters._
+import scala.concurrent.Future
+import scala.util.Failure
+import scala.util.Success
+import scala.util.Try
+
+import com.typesafe.scalalogging.LazyLogging
+
+object MessagingVerticle {
+  val KEY_TABLE_ID = "tableId"
+  val KEY_COLUMN_ID = "columnId"
+  val KEY_ROW_ID = "rowId"
+  val ID_KEYS = Seq(KEY_TABLE_ID, KEY_COLUMN_ID, KEY_ROW_ID)
+
+  val ADDRESS_CELL_CHANGED = "message.cell.change"
+  val ADDRESS_SERVICES_CHANGE = "message.services.change"
+  val ADDRESS_COLUMN_CREATED = "message.columns.created"
+  val ADDRESS_COLUMN_CHANGED = "message.columns.changed"
+  val ADDRESS_COLUMN_DELETED = "message.columns.deleted"
+
+  val ADDRESS_TABLE_CREATED = "message.tables.created"
+  val ADDRESS_TABLE_CHANGED = "message.tables.changed"
+  val ADDRESS_TABLE_DELETED = "message.tables.deleted"
+
+  val ADDRESS_ROW_DELETED = "message.rows.deleted"
+  val ADDRESS_ROW_CREATED = "message.rows.created"
+  val ADDRESS_ROW_ANNOTATION_CHANGED = "message.rows.annotation.changed"
+
+  val ADDRESS_CELL_ANNOTATION_CHANGED = "message.cells.annotation.changed"
+
+  val EVENT_TYPE_TABLE_CREATE = "table_create"
+  val EVENT_TYPE_TABLE_CHANGE = "table_change"
+  val EVENT_TYPE_TABLE_DELETE = "table_delete"
+  val EVENT_TYPE_ROW_CREATED = "row_create"
+  val EVENT_TYPE_ROW_DELETE = "row_delete"
+  val EVENT_TYPE_ROW_ANNOTATION_CHANGED = "row_annotation_changed"
+  val EVENT_TYPE_COLUMN_CREATE = "column_create"
+  val EVENT_TYPE_COLUMN_CHANGE = "column_change"
+  val EVENT_TYPE_COLUMN_DELETE = "column_delete"
+  val EVENT_TYPE_CELL_ANNOTATION_CHANGED = "cell_annotation_changed"
+  val EVENT_TYPE_CELL_CHANGED = "cell_changed"
```

Comment:
Mir ist aufgefallen, dass die events bzw addresses inkonsistent benamt sind. Deshalb hat auch erst mein manueller test nicht funktioniert.
In der doku heißt es z.B. event `cell_change`(Präsens), was ich auch richtig finde. Implementiert ist es aber als `cell_changed` (Präteritum)

Die addresses sind grundsätzlich eher im Präteritum, find ich auch gut. Der value sollte dann aber auch `message.cell.changed` heißen.